examples/src/components/data-view.tsx

```
@@ -0,0 +1,164 @@
+  if (typeof value === "boolean") {
+    return value ? "true" : "false";
+  }
+
+  return String(value);
+}
+
+/**
+ * A generic component that displays any data structure with proper formatting
+ * for BigInt, Uint8Array, Map, and other complex types.
+ */
+export default function DataView(props: {
```

Biome (2.1.2)

[error] Do not shadow the global "DataView" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)